homeassistant/components/starline/button.py

```diff
@@ -11,13 +11,32 @@
 from .const import DOMAIN
 from .entity import StarlineEntity
 
-BUTTON_TYPES: tuple[ButtonEntityDescription, ...] = (
+button_types = [
     ButtonEntityDescription(
         key="poke",
         translation_key="horn",
     ),
+    ButtonEntityDescription(
+        key="panic",
+        translation_key="panic",
+        entity_registry_enabled_default=False,
+    ),
+]
+
+button_types.extend(
+    [
+        ButtonEntityDescription(
+            key=f"flex_{i}",
+            translation_key="flex",
+            translation_placeholders={"num": str(i)},
+            entity_registry_enabled_default=False,
+        )
+        for i in range(1, 10)
+    ]
 )
 
+BUTTON_TYPES: tuple[ButtonEntityDescription, ...] = tuple(button_types)
```

Comment:
Why don't we just create 1 list directly? You can use `*[...]` to spread out a list in the list